internal/statemachine/image_definition.go

```diff
@@ -217,3 +220,42 @@ func newMissingURLError(context *gojsonschema.JsonContext, value interface{}, de
 type MissingURLError struct {
 	gojsonschema.ResultErrorFields
 }
+
+// generatePocketList returns a slice of strings that need to be added to
+// /etc/apt/sources.list in the chroot based on the value of "pocket"
+// in the rootfs section of the image definition
+func (ImageDef ImageDefinition) generatePocketList() []string {
+	pocketMap := map[string][]string{
+		"release": []string{},
+		"updates": []string{
+			fmt.Sprintf("deb http://archive.ubuntu.com/ubuntu/ %s-updates %s\n",
+				ImageDef.Series, strings.Join(ImageDef.Rootfs.Components, " "),
+			),
+			fmt.Sprintf("deb http://security.ubuntu.com/ubuntu/ %s-security %s\n",
+				ImageDef.Series, strings.Join(ImageDef.Rootfs.Components, " "),
+			),
+		},
+		"security": []string{
+			fmt.Sprintf("deb http://archive.ubuntu.com/ubuntu/ %s-updates %s\n",
```

Comment:
Wait, is this correct? Why are we adding the `updates` pocket when `security` is selected as the pocket? For security we should only have release + security.

Reply:
I was wondering about this myself. In our email thread you stated "They superset each other", which I thought meant security implies updates in the same way updates implies security. I'll make this change though, this is an easy one.

Reply:
Pushed an update to change this. Let me know if you think this is now correct.

Reply:
Looking good now!